syntax = "proto3";
package logtail.models;

message UntypedSingleValue {
    double Value = 1;
}

message MetricEvent {  
    uint64 Timestamp = 1;
    bytes Name = 2;  
    map<string, bytes> Tags = 3;
    oneof Value {
        UntypedSingleValue UntypedSingleValue = 4;
    }
}
